### ISSUE TITLE
Comments in COBOL

### ISSUE THREAD

**Issue by bitfhacker** [SonarSource/sonar-custom-rules-examples#31]
I don't know if this is the best place to post this. If not, please redirect me (/dev/null).

What it's the best way of validate comments in COBOL?
I want to validate the presence of tag MYTAG in comments in IDENTIFICATION DIVISION.

```
       IDENTIFICATION DIVISION.                                         00010000
       PROGRAM-ID.  DOW0003.                                            00020000
      *@<MYTAG>                                                         00030000
      *@      PROGRAM    : ABC00000                                     00040000
      *@      OBJECTIVE  : MODULO DE ACESSO A FICHEIROS DE SALDOS.      00050000
      *@      DATA       : MAIO 2018                                    00060000
      *@      AUTHOR     : JOHN DOE                                     00070000
      *@</MYTAG>                                                        00080000
       ENVIRONMENT DIVISION.                                            00090000
      **********************        
```

After exploring SSLR COBOL Toolkit, it seems that SSLR COBOL doesn't detect comments as a Node.

Can I validate COBOL comments with SSLR?
